examples/redwoodjs/package.json

```diff
@@ -14,7 +14,7 @@
     "extends": "@redwoodjs/eslint-config"
   },
   "engines": {
-    "node": ">=12",
+    "node": ">=16",
```

Comment:
This would still use Node 18, no?

Reply:
ack, forgot to push up that change; thanks!

https://github.com/vercel/vercel/pull/10395/commits/6ba9b2d7a3d429a8199222233d6eb1e3e5f39c07